File: JavaScript/8js 2/getRealFloor.js
/*
|--------------------------------------------------------------------------
| What's the real floor?
|--------------------------------------------------------------------------
|
*/

function getRealFloor(n) {
   return n - (n > 13 ? 2 : n > 0 ? 1 : 0);
}


function getRealFloor(n) {
   return n > 0 ? --n - (n >= 13) : n;
}


function getRealFloor(n) {
   return n > 0 ? n - 1 - (n > 12) : n;
}


console.log(getRealFloor(1), 0);
console.log(getRealFloor(5), 4);
console.log(getRealFloor(15), 13);
console.log(getRealFloor(-2), -2);

File: Python/8py 2/get_real_floor.py
"""
|--------------------------------------------------------------------------
| What's the real floor?
|--------------------------------------------------------------------------
|
"""

def get_real_floor(n):
   return n - (2 if n > 13 else 1 if n > 0 else 0)


def get_real_floor(n):
   if n<=0: return n
   if n<13: return n-1
   if n>13: return n-2


def get_real_floor(n):
   return n - (n > 0) - (n > 13)


print(get_real_floor(1), 0)
print(get_real_floor(5), 4)
print(get_real_floor(15), 13)